builder/vmware/iso/step_remote_upload.go

```diff
@@ -49,4 +50,27 @@ func (s *stepRemoteUpload) Run(state multistep.StateBag) multistep.StepAction {
 }
 
 func (s *stepRemoteUpload) Cleanup(state multistep.StateBag) {
+	if !s.DoCleanup {
+		return
+	}
+
+	driver := state.Get("driver").(vmwcommon.Driver)
+	// ui := state.Get("ui").(packer.Ui)
+
+	remote, ok := driver.(RemoteDriver)
+	if !ok {
+		return
+	}
+
+	path, ok := state.Get(s.Key).(string)
+	if !ok {
+		return
+	}
+
+	log.Printf("Cleaning up remote path: %s", path)
+	err := remote.remove(path)
```

Comment:
Could it use remote.Remove(path)?

Reply:
this PR is outdated and I'll need to open a new one